### ISSUE TITLE
Certbot might sometimes send incorrect content type (not application/jose+json)

### ISSUE THREAD

**Issue by schoen** [certbot/certbot#7341]
A user on the forum mentioned another content-type issue (which should be tracked separately, as it looks like it might be in the other direction with Boulder not understanding a content-type from Certbot rather than with Certbot not understanding a content-type from Pebble):

```
2019-08-14 10:25:18,749:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 “POST /acme/cert/redacted HTTP/1.1” 415 168
2019-08-14 10:25:18,750:DEBUG:acme.client:Received response:
HTTP 415
Server: nginx
Content-Type: application/problem+json
Content-Length: 168
Link: https://acme-v02.api.letsencrypt.org/directory;rel=“index”
Replay-Nonce: oksQitzt64My7oDIQhReG_G1DWKWUdJ901Yg89A9iMw
Expires: Wed, 14 Aug 2019 15:25:18 GMT
Cache-Control: max-age=0, no-cache, no-store
Pragma: no-cache
Date: Wed, 14 Aug 2019 15:25:18 GMT
Connection: close

{
“type”: “urn:ietf:params:acme:error:malformed”,
“detail”: "Invalid Content-Type header on POST. Content-Type must be “application/jose+json”",
“status”: 415
}
2019-08-14 10:25:18,755:DEBUG:acme.client:Error during a POST-as-GET request, your ACME CA may not support it:
urn:ietf:params:acme:error:malformed :: The request message was malformed :: Invalid Content-Type header on POST. Content-Type must be “application/jose+json”
```

_Originally posted by @schoen in https://github.com/certbot/certbot/issues/7339#issuecomment-523149846_

**Comment by bmw** [comment]
@schoen, do you know what version of `acme`/`certbot` they were using? I believe this was fixed 6 months ago in https://github.com/certbot/certbot/pull/6757.

**Comment by schoen** [comment]
@bmw, I asked about that even before filing this issue, but I haven't gotten a reply yet. But I think your guess is likely!

**Comment by schoen** [comment]
@bmw, it turns out that this user was using an OS-packaged 0.31.0, which predates the fix in #6757. So you were right.

This is a duplicate of #6756.
